Tests/RockLib.Logging.Tests/LogEntryTests.cs

```diff
@@ -227,6 +228,35 @@ public static void SetExtendedPropertiesMapsHashtableItemsWithStringKeysToExtend
         logEntry.ExtendedProperties.Count.Should().Be(2);
     }
 
+    [Fact]
+    public static void SetExtendedPropertiesWithTraceData()
+    {
+#if NET5_0_OR_GREATER || NETCOREAPP2_0_OR_GREATER
+        var logEntry = new LogEntry("Hello, world!", LogLevel.Info);
+
+#pragma warning disable CA2000 // Dispose objects before losing scope
+        var a = new Activity("foo");
```

Comment:
Probably can just add a using in front and I think the disable wouldn't be necessary anymore.

Reply:
In .NET Core 3.1 `System.Diagnositcs.Activity` class does not inherit `IDisposable`  so the compiler throws Error CS1674 when you try and put it in a using block.

Reply:
Could you make a `#if` for .NET Core 3.1 where you don't use a `using`, and then and `#else` where you do? Or vice-versa.

Reply:
I'm not sure what would be gained by doing that.  This is a unit test so I would think that testing each framework the same would be more important.